specs/TrackingPrevention.md

```diff
@@ -0,0 +1,197 @@
+    {
+        wil::com_ptr<ICoreWebView2Profile> profile;
+        CHECK_FAILURE(webView2_13->get_Profile(&profile));
+
+        auto profile5 = profile.try_query<ICoreWebView2Profile5>();
+        if (profile5)
+        {
+            profile5->put_TrackingPreventionUserPreference(value);
+        }
+    }
+}
+```
+
+# API Details
+```
+/// Tracking prevention user preference kind.
+[v1_enum] typedef enum COREWEBVIEW2_TRACKING_PREVENTION_USER_PREFERENCE_KIND {
+  /// Indicates that tracking prevention is turned off. Setting to this level if
+  /// `IsTrackingPreventionEnabled` is true means privacy protection features are enabled
+  /// but will not block cookie and resource requests from known tracking domains.
+  COREWEBVIEW2_TRACKING_PREVENTION_USER_PREFERENCE_KIND_OFF,
+  /// The least restrictive level of tracking prevention for users who enjoy
+  /// personalized advertisements and who do not mind being tracked on the web.
+  /// This only protects users against malicious trackers such as fingerprints and crypto miners.
+  COREWEBVIEW2_TRACKING_PREVENTION_USER_PREFERENCE_KIND_BASIC,
+  /// The default level of tracking prevention for users who want to see less creepy
+  /// advertisements that follow them around the web while they browse. Balanced aims to
+  /// block trackers from sites that users never engage with while minimizing the risk of
+  /// compatibility issues on the web.
+  COREWEBVIEW2_TRACKING_PREVENTION_USER_PREFERENCE_KIND_BALANCED,
+  /// The most restrictive level of tracking prevention for users who are okay
+  /// trading website compatibility for maximum privacy.
+  COREWEBVIEW2_TRACKING_PREVENTION_USER_PREFERENCE_KIND_STRICT,
+} COREWEBVIEW2_TRACKING_PREVENTION_USER_PREFERENCE_KIND;
+
+/// Additional options used to create WebView2 Environment.
+[uuid(12e494a2-c876-11eb-b8bc-0242ac130003), object, pointer_default(unique)]
+interface ICoreWebView2EnvironmentOptions3 : IUnknown {
+  /// The `IsTrackingPreventionEnabled` property is used to toggle tracking prevention feature in WebView2.
+  /// By default this feature is enabled which means privacy protection features are enabled that blocks cookies
+  /// and web requests from known tracking domains and set to
+  /// `COREWEBVIEW2_TRACKING_PREVENTION_USER_PREFERENCE_KIND_BALANCED`.
+  /// You can set this property to false to disable the tracking prevention feature that also skips the
+  /// related code and improves the performance.
+  ///
+  /// Tracking prevention protects users from online tracking by restricting
+  /// the ability of trackers to access browser-based storage as well as the network.
+  /// See [Tracking prevention](microsoft-edge/web-platform/tracking-prevention).
+  // MSOWNERS: monicach@microsoft.com
+  [propget] HRESULT IsTrackingPreventionEnabled([out, retval]  BOOL* value);
+  /// Sets the `IsTrackingPreventionEnabled` property.
+  // MSOWNERS: monicach@microsoft.com
+  [propput] HRESULT IsTrackingPreventionEnabled([in]  BOOL value);
```

Comment:
We have partners ask to provide an API to disable the feature that should also improve performance - we disable it when we create the webview to avoid the perf impact.

Also, we have asks from GitHub devs to change the level of tracking prevention from `Balanced` which is being the default currently to `Basic`. However, turning off after webview is startup doesn't avoid perf impact.

Reply:
I think that's clearer now in your doc updates. Thx